internal/commands/pull/command.go

```diff
@@ -258,17 +258,22 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
-	successfulTemplateWrites := make([]string, 0, len(clientTemplates))
```

Comment:
[question] i'm wondering if there was some magic previously that allowed this render properly, or was it just simply always rendering the list of templates horizontally and all within a single set of `[...]` brackets?

related to that, i'm surprised we do not have any tests for `pull/command_test.go` which verifies the output in the scenario where a template was written, can we add those?